nuitka/build/include/nuitka/helpers.h

```diff
@@ -247,6 +247,9 @@ extern PyObject *BUILTIN_TYPE3(PyObject *module_name, PyObject *name, PyObject *
 // For built-in built-in len() functionality.
 extern PyObject *BUILTIN_LEN(PyObject *boundary);
 
+// For built-in built-in any() functionality.
+extern PyObject *BUILTIN_ANY(PyObject *boundary);
```

Comment:
I don't think you ought to use "boundary" here. Not even sure it was a good name for `BUILTIN_LEN` then.

Reply:
You ought to match declaration and definition, I think `valuè` is fine indeed.

Reply:
Looks like length declaration and definition of `BUILTIN_LEN` is also different.